### PR TITLE
CLDR-16469 Change -2/-1 eras to inheritEras

### PR DESCRIPTION
**Pull request by sffc**
CLDR-16469

- [x] This PR completes the ticket.

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by macchiati** [comment]
Looking good. You'll probably need to add PathHeader information, etc as par https://cldr.unicode.org/development/updating-dtds. The tests should tell you what to do if they fail, for those.

Good catch on adjusting  !

**Comment by macchiati** [comment]
Added to agenda for tomorrow's meeting. I think this is a better solution than -1,-2; key will be whether we can get this into the beta & ICU w/o delaying the release.

File: docs/ldml/tr35-dates.md [hunk 1]
```diff
@@ -1046,10 +1049,25 @@ For a sequence of eras with specified start dates, the end of each era need not
...
+    <inheritEras calendar="gregorian" />
+    <eras>
+        <era type="0" start="645-6-19"/>
+        <era type="1" start="650-2-15"/>
+        <!-- ... -->
+    </eras>
+</calendar>
+```
+
+This means that the two eras from calendar "gregorian" should be inserted into the era list for "japanese" when formatting.
```
Comment (srl295) [inline docs/ldml/tr35-dates.md:1069]: The calculation and the formatting is inserted. I think this wording is great for now.

**Review by srl295 — APPROVED** [review]
Lgtm thanks !!

**Comment by srl295** [comment]
> Error:  (TestPathHeader.java:1315)  Error: //supplementalData/calendarData/calendar[@type="%A"]/inheritEras/_calendar: Unknown path header[], Supplemental	Unknown	calendarData	calendar[@type="japanese"]/inheritEras/_calendar
[823](https://github.com/unicode-org/cldr/actions/runs/4421856917/jobs/7753092216#step:16:824)


i'll take a look

**Comment by srl295** [comment]
@sffc  needs to target `maint/maint-43`

**Comment by srl295** [comment]
fix: 

```diff
diff --git a/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt b/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt
index d87e825f9e..7c1dfe3a61 100644
--- a/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt
+++ b/tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/PathHeader.txt
@@ -466,6 +466,7 @@
 //supplementalData/currencyData/region[@iso3166="%A"]/currency[@iso4217="%A"]/_tender                  ; Supplemental ; Currency ; $1 ; $2–-∞..∞; HIDE
 
 //supplementalData/calendarData/calendar[@type="%A"]/eras/era[@type="%A"]/_%E                  ; Supplemental ; Calendar ; $1 ; $2-$3 ; HIDE
+//supplementalData/calendarData/calendar[@type="%A"]/inheritEras/_calendar                     ; Supplemental ; Calendar ; $1 ; calendar_inheritEras ; HIDE
 //supplementalData/calendarData/calendar[@type="%A"]/calendarSystem/_type                      ; Supplemental ; Calendar ; $1 ; calendar_type ; HIDE
 
 //supplementalData/calendarPreferenceData/calendarPreference[@territories="%A"]/_ordering                      ; Supplemental ; Calendar ; Preference ; $1 ; HIDE
```

**Comment by srl295** [comment]
@sffc  sub-PR https://github.com/sffc/cldr/pull/4

**Review by macchiati — CHANGES_REQUESTED** [review]
Needs to target maint/maint-43

**Comment by sffc** [comment]
Replaced with #2790 on the maint branch

**Pull request closed without merging**